contents/javascript/Javascript-framework-Angular.md

```diff
@@ -0,0 +1,204 @@
+<frontmatter>
+  title: Introduction to Angular
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to Angular
+
+Author(s): [Ronak Lakhotia](https://github.com/RonakLakhotia)
+
+## What Is Angular?
+
+Angular is an open-source frontend Javascript framework that helps you build <a href="https://www.bloomreach.com/en/blog/2018/07/what-is-a-single-page-application.html"> Single Page Web Applications</a>.
+The library provides a number of features that make it trivial to implement the complex requirements of modern applications, such as data binding, routing, and animations. Angular is maintained by Google, as well as a community of individual developers
+
+>Angular started as a side project. Back in 2009, Miško Hevery and Adam Abrons released a project that would help developers, as well as designers, build web applications using simple HTML tags. The name “Angular” comes from the angle brackets, or < >, that surround all HTML tags.
+>
+>Miško described the idea behind the framework in an <a href="https://www.infoworld.com/article/2612801/javascript/whats-so-special-about-googles-angularjs.html">interview done in 2013</a>. 
+
+With Angular, designers can use HTML as the template language and it allows for the extension of HTML's syntax to convey the application's components effortlessly. Angular makes much of the code you would otherwise have to write, completely redundant.
+
+## What are the different versions of Angular?
+
+AngularJS or Angular1 is a JavaScript front-end framework developed by Google whereas Angular is a TypeScript-based open-source front-end web application platform led by the Angular Team at Google.
+There have been 5 versions of Angular so far, which are AngularJS, Angular2, Angular4, Angular5 and the latest one being Angular6.
+There are key differences between the various versions as Google tried to enhance the tool given the fierce competition from React and Vue. 
+More details about the differences can be found in this [Quora](https://www.quora.com/What-are-the-different-versions-of-AngularJS) post.
```

Comment:
Why would a reader be concerned about different versions on Angular so early in the article?